src/testFixtures/groovy/grails/plugin/geb/WebDriverContainerHolder.groovy

```diff
@@ -98,14 +98,11 @@ class WebDriverContainerHolder {
         }
 
         currentConfiguration = specConfiguration
-        currentContainer = new BrowserWebDriverContainer()
-        if (grailsGebSettings.recordingEnabled) {
```

Comment:
It looks like you always start the recording container

Reply:
@sbglasius If `grailsGebSettings.recordingMode` is set to `SKIP`, `grailsGebSettings.isRecordingEnabled()` will return `false` and the `recordingMode` on the `currentContainer` will not be changed. The default `WebDriverContainer.recordingMode` is `RECORD_FAILING`.

With this change, if the `grailsGebSettings.recordingMode`  is `SKIP`, the `WebDriverContainer.recordingMode` will also be `SKIP`